docs/fields/field-details.asciidoc

```diff
@@ -8660,6 +8660,33 @@ example: `background-tasks`
 
 // ===============================================================
 
+|
+[[field-service-node-roles]]
+<<field-service-node-roles, service.node.roles>>
+
+a| Roles of a service node.
+
+This allows for distinction between different running roles of the same service.
+
+In the case of Kibana, the `service.node.role` could be `ui` or `background-tasks` or both.
```

Comment:
We ended up changing from `background-tasks` to `background_tasks` in the Kibana PR for consistency with ES: https://github.com/elastic/kibana/pull/135272#discussion_r908530524

So maybe we should change the examples in this PR to match

Reply:
Agreed!

Reply:
suggestion applied